webview-ui/src/components/ResolutionsPage/ReceivedMessage.tsx

```diff
@@ -1,15 +1,49 @@
 import React from "react";
-import { FlexItem, Label } from "@patternfly/react-core";
+import { Message } from "@patternfly/chatbot";
+import rehypeRaw from "rehype-raw";
+import rehypeSanitize from "rehype-sanitize";
+import botAv from "./bot_avatar.svg";
```

Comment:
```suggestion
import botAv from "./bot_avatar.svg?inline";
```

Force vite to inline the svg and resolve any headaches where the extension can't load the image. https://vite.dev/guide/assets#explicit-inline-handling